src/app/episode/episode.consts.ts

```diff
@@ -0,0 +1,8 @@
+import { changeInfoPropertiesAction } from './info/store/info.actions';
+
+/**
+ * Store name where all episode data is found
+ */
+export const EPISODE_STORE = '@flame/episode';
```

Comment:
I'm creating this pattern so we can store the Store names for each domain with `@flame/***`, so it's easy to spot from model names.

Mostly to avoid confusion in the Store having:  "episode -> episode -> info"